package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "core-data-modelling",
+  "version": "0.0.1",
+  "description": "Get Qlik Associative Engine data model insights.",
+  "author": "QlikTech International AB",
+  "license": "MIT",
+  "scripts": {
+    "build": "cross-env NODE_ENV=production parcel build --no-cache --experimental-scope-hoisting src/index.html",
+    "start": "parcel src/index.html",
+    "lint": "eslint --ext .js,.jsx .",
+    "lint:fix": "eslint --ext .js,.jsx --fix ."
+  },
+  "dependencies": {
+    "enigma.js": "^2.3.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.5.0",
+    "react-dom": "^16.4.1",
+    "react-outside-click-handler": "^1.2.2",
+    "node-sass": "^4.9.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-eslint": "10.0.1",
+    "babel-plugin-transform-async-to-generator": "6.24.1",
```

Comment:
https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-async-to-generator indicates there's a babel v7-compatible version out there instead of this.